creator-node/src/services/sync/primarySyncFromSecondary.js

```diff
@@ -185,11 +195,34 @@ async function _primarySyncFromSecondary({
         throw e
       }
 
-      const clockInfo = fetchedCNodeUser.clockInfo
-      if (clockInfo.localClockMax <= clockInfo.requestedClockRangeMax) {
+      /**
+       * TODO update this once all nodes are running 0.3.66
+       *
+       * cnodeUser.clock field is used for comparison since it is max(requestedClockRangeMax, actual clockMax)
+       * This means:
+       *    cnodeUser.clock < requestedClockRangeMax
+       *      - no more data to fetch
+       *      - completed = true
+       *    cnodeUser.clock = requestedClockRangeMax
+       *      - may or may not be more data to fetch
+       *      - completed = false
+       *    cnodeUser.clock > requestedClockRangeMax
+       *      - this never happens
+       */
+      if (fetchedCNodeUserClockVal < requestedClockRangeMax) {
```

Comment:
the main change here is to use `cnodeUser.clock` instead of `cnodeUser.clockInfo.localClockMax`

i know how shitty this code is lol